i18n/fr/docusaurus-plugin-content-docs/version-22.04/graph-views/map-web-update.md

```diff
@@ -32,9 +32,37 @@ sudo yum update centreon-map-engine centreon-map-web-client
 </TabItem>
 <TabItem value="Debian 11" label="Debian 11">
 
-``` shell
-sudo apt update centreon-map-engine centreon-map-web-client
-```
+ - Si MAP seul est installé :
+ 
+ ``` shell
+ sudo apt upgrade centreon-map-engine centreon-map-web-client
+ ```
+  
+ - Si MAP et MAP Legacy sont installés sur le même serveur :
+   
+   - Faites une sauvegarde du fichier **map.cnf** :
+    
+    ```shell
+    cp /etc/mysql/map.cnf /etc/mysql/map.cnf.bk
```

Comment:
The correct path for map.cnf on Debian is `/etc/my.cnf.d/map.cnf`
so the backup copy should be `cp /etc/my.cnf.d/map.cnf /etc/my.cnf.d/map.cnf.bk`